src/main/kotlin/com/team4099/robot2021/Robot.kt

```diff
@@ -46,10 +50,12 @@ object Robot : TimedRobot() {
 
   override fun teleopInit() {
     autonomousCommand.cancel()
+    ZeroSensorsCommand()

```

Comment:
do we want to schedule this bc just creating it won't do anything

also zero should def not happen at the start of teleop